contrib/jemalloc-cmake/CMakeLists.txt

```diff
@@ -34,9 +34,9 @@ if (OS_LINUX)
     # avoid spurious latencies and additional work associated with
     # MADV_DONTNEED. See
     # https://github.com/ClickHouse/ClickHouse/issues/11121 for motivation.
-    set (JEMALLOC_CONFIG_MALLOC_CONF "percpu_arena:percpu,oversize_threshold:0,muzzy_decay_ms:5000,dirty_decay_ms:5000")
+    set (JEMALLOC_CONFIG_MALLOC_CONF "percpu_arena:percpu,oversize_threshold:0,muzzy_decay_ms:0,dirty_decay_ms:5000")
```

Comment:
I'm fine with the change, let's also update the comment above to reflect this change.

P.S. I've made some benchmark here for MADV_FREE vs MADV_DONTNEED - https://gist.github.com/azat/aad8c509355970e3f09fe90279f7825d, in short it is 37% slower, but I doubt that this is significant for this code path.